### DISCUSSION TITLE
These six distant galaxies captured by JWST are wowing astronomers

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#89] [Astrophysics and Physics]
Nature NEWS 30 June 2023

See researchers’ favourites from a survey of the deep Universe by the James Webb Space Telescope.

[https://www.nature.com/articles/d41586-023-02157-9](https://www.nature.com/articles/d41586-023-02157-9)

**Comment by OlliSant** [comment]
They are all big, developed, old galaxies, otherwise we couldn't see them at all. We must compare the farthest galaxies in Hubble, and how they look now, and what is said of them then. Are there such studies?

**Reply by RedshiftDrift** [reply]
There are many such studies, see e.g. the papers listed here [https://cosmology.info/resources/marmet_l/reviewed-publications.html#5-old-systems](https://cosmology.info/resources/marmet_l/reviewed-publications.html#5-old-systems)

**Reply by OlliSant** [reply]
Thanks, Louis
